images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -0,0 +1,50 @@
+Import-Module (Join-Path "..\SoftwareReport" "SoftwareReport.Android.psm1") -DisableNameChecking
+
+Describe "Android SDK" {
+    $androidToolset = (Get-ToolsetContent).android
+    $androidInstalledPackages = Get-AndroidInstalledPackages
+
+    $platformTestCases = @()
+    $platformList = $androidToolset.platform_list
+    $platformList | ForEach-Object {
+        $platformTestCases += @{ platformVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $buildToolsTestCases = @()
+    $buildToolsList = $androidToolset.build_tools
+    $buildToolsList | ForEach-Object {
+        $buildToolsTestCases += @{ buildToolsVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $extraPackagesTestCases = @()
+    $extraPackagesList = $androidToolset.extra_list
+    $extraPackagesList | ForEach-Object {
+        $extraPackagesTestCases += @{ extraPackage = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $addonsTestCases = @()
+    $addonsList = $androidToolset.addon_list
+    $addonsList | ForEach-Object {
+        $addonsTestCases += @{ addonPackage = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    It "Platform version <platformVersion> is installed" -TestCases $platformTestCases {
+        $regex = "platforms;$platformVersion"
+        [boolean]($installedPackages | Where-Object { $_ -match $regex }) | Should -BeTrue
```

Comment:
Does it make sense to use -Match parameter?

`$installedPackages | Should -Match "platforms;$platformVersion"`

Reply:
Done, thank you!